File: Public/Scripts/Context/Channel/EditMessageContext.ts
export class EditMessageContext
{
    public ChannelId: number;
    public MessageId: number;
    public Message: string | null;

    constructor(
        channelId: number,
        messageId: number,
        message: string | null,
    ) {
        this.ChannelId = channelId;
        this.MessageId = messageId;
        this.Message = message;
    }
}

File: Internal/Context/Channel/EditMessageContext.cs
using System.Text.Json.Serialization;

namespace BonfireServer.Internal.Context.Channel;

[Serializable]
public class EditMessageContext : IBaseContext
{
    [JsonIgnore] public string? Token { get; set; }
    public string? Message { get; set; }
    public long ChannelId { get; set; }
    public long MessageId { get; set; }
}